_data/collection-index.yml

```
@@ -1,3 +1,8 @@
+- name: Bob Build System
+  maintainer: Dirk Louwers
+  contact: https://github.com/dlouwers/devcontainer-feature-bob/issues
+  repository: https://github.com/dlouwers/devcontainer-feature-bob
+  ociReference: ghcr.io/dlouwers/devcontainer-feature-bob/bob
```

```suggestion
  ociReference: ghcr.io/dlouwers/devcontainer-feature-bob
```

The `ociReference` here refers to the collection metadata artifact instead of the individual Features. Hence, the author needs to only add a single entry irrespective of the number of Features that might exist in their repository.

Can you help update this? thanks!

---

Updated the reference to point to the root of the features and updated the title to reflect this